spec/core/ics-004-channel-and-packet-semantics/UPGRADES.md

```diff
@@ -32,12 +33,12 @@ enum ChannelState {
   TRYOPEN,
   OPEN,
   INITUPGRADE,
-  TRYUPGRADE,
+  BLOCKUPGRADE,
```

Comment:
In general I like the `BLOCK` state naming, but I could see it being mildly confusing (block upgrade, sounds like you are halting the upgrade, which is partially true). Just curious if there's any alternatives we should consider?

Reply:
Do we need to change this enum at all? I feel like it could remain as TRYUPGRADE (consistent with other handshakes) but the spec can outline that being in this state blocks new packets.

Reply:
Semantically the upgrade steps are different then the opening handshake. It's no longer 4 steps, it is either 5 or 6:
chainA, chainB
init
block, block
open, open

or 
init, init
block, block
open, open

It is required both sides enter into the block state, whereas in the opening handshake you can skip over try (init -> open)

Reply:
is this still the latest version? Or are we removing the BLOCKUPGRADE state?

Reply:
No, we realized the above had an issue. I've added two new states to the ChannelState enum: `TRYUPGRADE` and `ACKUPGRADE`.

And then a separate `FLUSHSTATUS`. Please implement only the ChannelState enum changes for now.

We will deal with FLUSHSTATUS in a future PR